backend/molgenis-emx2/src/main/java/org/molgenis/emx2/Column.java

```diff
@@ -608,4 +612,10 @@ public void setRefLink(String refLink) {
   public boolean isPrimaryKey() {
     return getKey() == 1;
   }
+
+  @Override
+  public int compareTo(Object o) {
```

Comment:
i think you can do Comparable<Column> and need a equals and hashcode impl (for use in certain sorted collections)( or use a record)

Reply:
I don't want to do a full compare but only 'sort'. I don't believe you need equals and hashcode for that.

However, could instead, not use Comparable at all and instead in the places where I sort specifically implement comparator. I suppose that might be better.

Reply:
yes the issue is that if you override the compareTo it gets used by the collections

Reply:
Made some improvements